src/frames.ts

```diff
@@ -730,6 +730,26 @@ export class Frame {
         (handle, deadline) => handle.focus());
   }
 
+  async textContent(selector: string, options: types.TimeoutOptions = {}): Promise<string> {
+    return await this._retryWithSelectorIfNotConnected('textContent', selector, options,
+        (handle, deadline) => handle.textContent() as Promise<string>);
+  }
+
+  async innerText(selector: string, options: types.TimeoutOptions = {}): Promise<string> {
+    return await this._retryWithSelectorIfNotConnected('innerText', selector, options,
+        (handle, deadline) => handle.innerText() as Promise<string>);
```

Comment:
Why don't we update ElementHandle.innerText() and ElementHandle.innerHTML() to return string as well?

Reply:
done